composite/style/shell/action.yml

```diff
@@ -20,6 +20,8 @@ runs:
           for sh_file in ${{ inputs.changed_shell_files }}; do
             shellcheck -f diff $sh_file | patch -p1
           done
+
+          rm -rf vendor/
```

Comment:
maybe put a comment on why this is here else people might freak out. I know I will once I lose context, so next week :)

Reply:
Done!